HISTORY.md

```diff
@@ -4,6 +4,14 @@ In the beginning, programmers created the databases. Now the databases were form
 
 And so [Salesforce.org](http://salesforce.org/) said “Let there be data,” and there was Snowfakery. And it was good.
 
+## Snowfakery 2.4
+
+The Salesforce load order now takes into account the order in which
+object templates were declared in a recipe. The other considerations
```

Comment:
```suggestion
object template declarations in the recipe. The other considerations
```

Reply:
I believe there's one recipe when you're running Snowfakery and that it now determines the load order. If I'm wrong, my suggestions are likely wrong too. Let me know.